tools/impeller.gni

```diff
@@ -190,84 +193,85 @@ template("impeller_shaders_real") {
       args += [ "--ios" ]
     }
   }
+  if (is_mac || is_ios) {
```

Comment:
Would it work to wrap all of `impeller_shaders_real` inside of `if (impeller_shaders_supports_platform)` ?

Reply:
In this case yes .. I added that after doing this. I think we'll want to do some kind of clean up for this though. I imagine we'll eventually split these into separate templates per platform/target...

Reply:
Yeah this will definitely need to be refactored when adding OpenGL and/or Vulkan.

Reply:
Ahh I can just drop this because it's not invoked unless `impeller_shaders_supports_platform`.

Reply:
Also we really should fix impellerc depending on Dart. I'm pretty sure that's all because of tracing via FML...